azure-pipelines.yml

```diff
@@ -807,6 +807,12 @@ jobs:
           artifact: rpm-ubi9-${{ arch.value }}
           python_package: python3.12
           pool: ${{ arch.pool }}
+        Red Hat Universal Base Image 10 ${{ arch.name }}:
+          dockerfile: ubi
+          image: registry.access.redhat.com/ubi10/ubi:10.0
```

Comment:
Red Hat use `10.0` tag again. It uses weird `9.0.0` tag in RHEL 9: https://github.com/Azure/azure-cli/pull/23556/files#r952102485

There are so many inconsistencies in RHEL.